generators/app/templates/bin/setup

```diff
@@ -1,34 +1,14 @@
-#!/usr/bin/env ruby
-require 'pathname'
-require 'fileutils'
-include FileUtils
+#!/bin/bash
+function realpath { echo $(cd $(dirname $1); pwd)/$(basename $1); }
+SCRIPT=`realpath $0`
+ROOT_PATH="$(dirname $SCRIPT)/../"
 
-# path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+echo "Server setup..."
+cd "$ROOT_PATH/server" && bin/setup
 
-def system!(*args)
-  system(*args) || abort("\n== Command #{args} failed ==")
-end
+wait
 
-chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file.
+echo "Client setup..."
+cd "$ROOT_PATH/client" && npm install
```

Comment:
Yeah, but we don't have any easy way of showing a nice default for the user and asking if he wants to change it or no withing the setup. That's why I ended up using yeoman, because it already has an easy way to do all these things.